dell/inspiron/7405/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, ... }:
+
+with lib;
+
+{
+  imports = [
+    ../../../common/cpu/amd
+    ../../../common/pc/laptop
+    ../../../common/pc/laptop/ssd
+  ];
+
+  # TSC is unstable
+  boot.kernelParams = [ "notsc" "trace_clock=local" ];
+
+  hardware.enableRedistributableFirmware = mkDefault true;
+
+  services.xserver = {
+    # Touchpad
+    libinput.enable = mkDefault true;
+
+    # No nvidia driver
+    videoDrivers = mkDefault [
+      "amdgpu"
+      "radeon"
+      "fbdev"
+    ];
```

Comment:
Two quick thoughts on `videoDrivers`:

* Are you sure the older `"radeon"` driver supports this gpu? 
(I think it's not listed under "Kernel Drivers" in https://linux-hardware.org/?id=pci:1002-1636-1002-1636 if I interpreted that correctly)

* Would it make sense to import `../../../common/gpu/amd`? I see some other configs do that, I'm not sure what is considered best practice here.

Reply:
Good points.

Reply:
> Are you sure the older "radeon" driver supports this gpu?

No, it uses amdgpu. Now that you pointed it out I'm surprised how I didn't notice this earlier lol.

> Would it make sense to import ../../../common/gpu/amd?

I thought it was for discrete gpus as nearly every laptop config I saw doesn't import gpu modules.

Reply:
> I thought it was for discrete gpus as nearly every laptop config I saw doesn't import gpu modules.

Does this laptop have integrated graphics? 

E.g. I know on macbook pro 11-5, it has `i915` for intel integrated graphics and the `amdgpu` driver for discrete graphics.

Reply:
It has only integrated graphics. I have imported `../../../common/gpu/amd` and it worked without issues.